src/lib/api/configuration-shell-api/configurationShellApi.ts

```diff
@@ -44,7 +44,7 @@ export class ConfigurationShellApi {
         bearerToken: string,
         value: string,
         settingsType: string,
-        use_authentication: boolean = false,
+        use_authentication: boolean = true,
```

Comment:
In our Readme, we define the authentication default value as "false" (see AUTHENTICATION_FEATURE_FLAG). Why do we need this change here?

Reply:
adapted the usage of the feature flag